docs/msf/managing-store-configuration.mdx

```diff
@@ -79,17 +100,26 @@ Accept: application/json
 }
 ```
 
+</Tab>
+</Tabs>
+
 Because a custom store name exists for channel 123, the channel-specific store name overrides the global default. Shoppers who interact with this channel will see the custom store name at each place it appears in the channel. When the global default `store_name` in the merchant's store profile settings changes, this channel will not reflect that change. However, any other changes to the global default store profile will affect channel 123, because `store_name` is the only channel-specific value defined. 
 
 The following request-response pair shows an example of a channel with a complete set of channel-specific store profile settings configured.
 
-```http title="Example request: Get store profile settings for channel 124"
+<Tabs items={['Request', 'Response']}>
+<Tab>
+
+```http filename="Example request: Get store profile settings for channel 124"
 GET https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/settings/profile?channel_id=124
 X-Auth-Token: {{ACCESS_TOKEN}}
 Accept: application/json
 ```
-&nbsp;
-```json title="Example response: Get store profile settings for channel 124" lineNumbers
+
+</Tab>
+<Tab>
+
+```json filename="Example response: Get store profile settings for channel 124" showLineNumbers
```

Comment:
```suggestion
```json filename="Example response: Get store profile settings for channel 124" showLineNumbers copy
```